pkgs/development/misc/resholve/default.nix

```diff
@@ -84,7 +82,10 @@ let
       "*XMLRPC*"
     ];
     enableOptimizations = false;
-  };
+  } // lib.optionalAttrs (!(stdenv.isAarch64 && stdenv.isDarwin)) {
```

Comment:
I think we can have it as non-conditional, it is not like adding them will increase the derivation size too much.